Script_Source/Device/README.md

```diff
@@ -0,0 +1 @@
+# Device Source Files
```

Comment:
We currently do not have a folder called Device. Can these scripts go into an existing folder?

Reply:
Also, the scripts need to go into their own sub folder with a readme.md file. This is needed so that the automation will work to create zip file.